example/scene.ts

```diff
@@ -45,6 +45,6 @@ export const VIEWER: ViewerProperty = {
   },
   terrain: {
     enabled: true,
-    normal: true,
+    normal: false,
```

Comment:
Why do we need to disable this? This is necessary to check the image based lighting for globe.
```suggestion
    normal: true,
```

Reply:
Updated! I just feel it looks dark at night and removed it when test and forgot to revert it.